WordPress/Classes/ViewRelated/Reader/WPImageViewController.m

```diff
@@ -568,6 +569,14 @@ - (void)setupAccessibility
 {
     self.imageView.isAccessibilityElement = YES;
     self.imageView.accessibilityTraits = UIAccessibilityTraitImage;
+    
+    if (self.media != nil && self.media.title != nil) {
+        self.imageView.accessibilityLabel = [NSString stringWithFormat:NSLocalizedString(@"Fullscreen view of image %@. Double tap to dismiss the screen", @"Accessibility label for when image is shown to user in full screen, with instructions on how to dismiss the screen. Paramater is the title of the image"), self.media.title];
```

Comment:
Nitpick typo: `Paramater` -> `Parameter` (or `placeholder` might be the more common term we use  🤔)

I think we could also trim these messages slightly to simply say "Double tap to dismiss" at the end (removing `the screen`).